### PR TITLE
Rename "middle" -> "mid"

### PR DESCRIPTION
**Pull request by olilarkin**
I think the mid frequency EQ knob would be better named "mid" rather than "middle"

### CODE REVIEW

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
   activations::Activation::enable_fast_tanh();
   GetParam(kInputLevel)->InitGain("Input", 0.0, -20.0, 20.0, 0.1);
   GetParam(kToneBass)->InitDouble("Bass", 5.0, 0.0, 10.0, 0.1);
-  GetParam(kToneMid)->InitDouble("Middle", 5.0, 0.0, 10.0, 0.1);
+  GetParam(kToneMid)->InitDouble("Mid", 5.0, 0.0, 10.0, 0.1);
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:63]: Would this mess with people's saved presets?

Comment (olilarkin) [reply NeuralAmpModeler/NeuralAmpModeler.cpp:63]: no

**Comment by James-F2** [comment]
I beg to differ, please keep the full word "Middle" if you also keep "Bass" and "Treble" like on Marshalls.
Unless you go for "Low" "Mid" "High" which are on other amps like the 5150.
I hope it makes sense.

M

**Comment by olilarkin** [comment]
Makes sense.

**Comment by sdatkinson** [comment]
@James-F2 thanks for the references 🙂

**Pull request closed without merging**